internal/jwtauth/path_config_test.go

```diff
@@ -4,38 +4,35 @@ import (
 	"context"
 	"reflect"
 	"testing"
-	"time"
 
 	"github.com/go-test/deep"
-	log "github.com/hashicorp/go-hclog"
-	"github.com/hashicorp/vault/sdk/helper/logging"
 	"github.com/hashicorp/vault/sdk/logical"
 )
 
 func TestConfig_Write(t *testing.T) {
 	t.Parallel()
 	backend, storage := createTestBackend(t)
 
-	data := testConfig()
+	configData := testConfig()
```

Comment:
How come this change comes in again? Maybe a separate PR?

Reply:
We only changed _one_ of the two locations last time this one came up. Can create a separate PR, but since it was test-reladed, I included it here.